package.json

```diff
@@ -1,15 +1,12 @@
 {
   "name": "@ledgerhq/wallet-common",
   "version": "0.13.0",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "license": "MIT",
   "scripts": {
-    "preinstall": "node -e \"if (process.env.npm_execpath.indexOf('yarn') === -1) { console.log('\u001b[31mPlease use yarn\u001b[0m'); process.exit(1); }\"",
     "flow-typed": "flow-typed install -s",
-    "build": "yarn clean && yarn documentation && babel --source-maps -d lib src && flow-copy-source src lib",
```

Comment:
why no longer building the documentation?

Reply:
I think it's better to clearly separate concerns rather than a big "do everything" button. We can still add a "build-lib-and-documentation" script if we want.

Reply:
so maybve just putting it in `prepare`, or whatever the script to be called for the publish

Reply:
the problem with `prepare` is the fact it is *always* triggered when installing/reinstalling deps with `yarn` ( = `yarn install`), which is highly annoying.